### PR TITLE
[Runtime-1061] Collection runner: Added iterationData methods

### PR DESCRIPTION
**Pull request by arnitkun**
This PR adds pm.iterationData methods under "running multiple iterations" page, describing all the methods accessible by the iterationData object that can be used during collection runs.

### CODE REVIEW

**Comment by arnitkun** [comment]
Changes made at both places, mentioned iterationData and explained the methods in sandbox API reference.

File: v6/postman/collection_runs/running_multiple_iterations.md [hunk 1]
```diff
@@ -45,6 +46,12 @@ Iterations in the collection runner are 1-indexed with the first iteration begin
 
 Note that this is different than the iteration count accessible programmatically in the [Postman sandbox](/docs/v6/postman/scripts/postman_sandbox_api_reference), which is 0-indexed with the first iteration beginning with a count of zero.
 
+## Programmatically customize iteration data
+
+To provide data for a collection run, the Collection runner provides a "Data file" option. However if one wants to access and manipulate that data as the collection runs, it must be done programmatically. This can be done by the `pm.iterationData` object, which provides several methods to programmatically access and manipulate the data, allowing the access of iteration data during a collection run.
+
+For a list of methods provided by the iterationData object, see: [pm.iterationData](/Users/postman/postman-docs/v6/postman/scripts/postman_sandbox_api_reference.md)
```
Comment (kaustavdm) [inline v6/postman/collection_runs/running_multiple_iterations.md]: Why did you link this to a local file? This should be `/docs/postman/scripts/postman_sandbox_api_reference`.

File: v6/postman/scripts/postman_sandbox_api_reference.md [hunk 2]
```diff
@@ -212,7 +212,17 @@ The `iterationData` object contains data from the data file provided during a co
 
 * `pm.iterationData.get(variableName:String):function → *`
 * `pm.iterationData.toObject():function → Object`
-
+* `pm.iterationData.addLayer(list: VariableList):function → void`
```
Comment (kaustavdm) [inline v6/postman/scripts/postman_sandbox_api_reference.md]: Need a one line description of what each of these methods do. Add a description for each of these.

Comment (arnitkun) [reply v6/postman/scripts/postman_sandbox_api_reference.md]: Added descriptions for methods in sandbox api reference.

File: v6/postman/collection_runs/running_multiple_iterations.md (hunk 1 above)
Comment (arnitkun) [reply v6/postman/collection_runs/running_multiple_iterations.md]: fixed.

File: v6/postman/collection_runs/running_multiple_iterations.md [hunk 3]
```diff
@@ -45,6 +46,12 @@ Iterations in the collection runner are 1-indexed with the first iteration begin
 
 Note that this is different than the iteration count accessible programmatically in the [Postman sandbox](/docs/v6/postman/scripts/postman_sandbox_api_reference), which is 0-indexed with the first iteration beginning with a count of zero.
 
+## Programmatically customize iteration data
+
+To provide data for a collection run, the Collection runner provides a "Data file" option. However if one wants to access and manipulate that data as the collection runs, it must be done programmatically. This can be done by the `pm.iterationData` object, which provides several methods to programmatically access and manipulate the data, allowing the access of iteration data during a collection run.
+
+For a list of methods provided by the iterationData object, see: [pm.iterationData](docs/v6/postman/scripts/postman_sandbox_api_reference.md)
```
Comment (kaustavdm) [inline v6/postman/collection_runs/running_multiple_iterations.md]: Missing a `/` at the beginning of URL. Do not use `.md` at the end.

File: v6/postman/collection_runs/running_multiple_iterations.md [hunk 4]
```diff
@@ -45,6 +46,12 @@ Iterations in the collection runner are 1-indexed with the first iteration begin
 
 Note that this is different than the iteration count accessible programmatically in the [Postman sandbox](/docs/v6/postman/scripts/postman_sandbox_api_reference), which is 0-indexed with the first iteration beginning with a count of zero.
 
+## Programmatically customize iteration data
+
+To provide data for a collection run, the Collection runner provides a "Data file" option. However if one wants to access and manipulate that data as the collection runs, it must be done programmatically. This can be done by the `pm.iterationData` object, which provides several methods to programmatically access and manipulate the data, allowing the access of iteration data during a collection run.
```
Comment (kaustavdm) [inline v6/postman/collection_runs/running_multiple_iterations.md]: Use active voice. "However, if you want to access and manipulate that data in the collection runs, you will need to it programmatically through scripts."

Comment (arnitkun) [reply v6/postman/collection_runs/running_multiple_iterations.md]: Done.

File: v6/postman/collection_runs/running_multiple_iterations.md (hunk 3 above)
Comment (arnitkun) [reply v6/postman/collection_runs/running_multiple_iterations.md]: fixed the path. It seems I was confused about the path of the file as it is placed on the web server. I should have just used the other references.

**Pull request merged**